pkg/tar/tar_test.go

```diff
@@ -236,6 +278,18 @@ func verifyDirectory(t *testing.T, dir string, files []fileDesc) {
 					t.Errorf("File content is not equal for %s. Expected: %s, Actual: %s\n",
 						relpath, fd.content, content)
 				}
+				if isSymLink(fd.mode) {
+					target, err := os.Readlink(path)
+					if err != nil {
+						return err
+					}
+					if target != fd.target {
+						t.Errorf(
+							"Symbolic link %q points to wrong path. Expected: %s, Actual: %s\n",
+							fd.name, fd.target, target,
+						)
```

Comment:
maybe you could do here the same as in [here](https://github.com/openshift/source-to-image/pull/307/files#diff-203bceaa948b8f85722a4d86502613f6R201)?